pom.xml

```diff
@@ -331,7 +331,7 @@
 		      	<dependency>
 			        <groupId>com.google.guava</groupId>
 			        <artifactId>guava</artifactId>
-			        <version>15.0</version>
+			        <version>33.4.8-jre</version>
```

Comment:
may be failed on release stage
use version=15.0
https://mvnrepository.com/artifact/org.sonatype.plugins/nexus-staging-maven-plugin/1.7.0

Reply:
This step used in release workflow only